index.html

```diff
@@ -2811,7 +2823,8 @@ <h2>JSON-LD</h2>
         <h3>Production</h3>
 
         <p>
-The <a>DID document</a> and any DID document data structures expressed by the <a href="#data-model">data model</a> MUST
+The <a>DID document</a>, DID document data structures, and
+representation-specific entries <a data-cite="INFRA#maps">map</a> MUST be MUST
```

Comment:
```suggestion
representation-specific entries <a data-cite="INFRA#maps">map</a> MUST be
```

Reply:
`MUST be MUST be` becomes `MUST be be` ... so take my suggestion below instead

Reply:
Done.